public/kubernetes-guides/advanced-guides/kuberay.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "KubeRay"
+description: "Install Ray on Kubernetes."
+---
+
+import { version } from '/snippets/custom-variables.mdx';
+
+[Ray](https://www.ray.io/) is a project for running machine learning jobs in a cluster of machines. [KubeRay](https://github.com/ray-project/kuberay) is an operator for installing Ray on top of Kubernetes.
+For up-to-date installation guide check with the [upstream Ray documentation](https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/kuberay-operator-installation.html).
+
+## Install Ray operator with helm
+
+Create a Kubernetes cluster with [`talosctl`](https://docs.siderolabs.com/omni/getting-started/how-to-install-talosctl) or via [Omni](https://docs.siderolabs.com/omni/overview/what-is-omni).
```

Comment:
relative URLs  just incase we change docs.siderolabs.com in the future, so that the links are not broken